.github/CI.yml

```
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: gradle
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Build with Gradle
+        run: ./gradlew clean build --no-daemon
+
+      - name: GOMS Backend CI success Discord Notification
+        uses: sarisia/actions-status-discord@v1
+        if: ${{ success() }}
+        with:
+          title: ✅ GOMS-Backend-CI 성공! ✅
+          webhook: ${{ secrets.DISCORD_WEBHOOK_CI_BOT }}
```

저희는 CI, CD 둘 다 같은 디스코드 채널에서 하니까 환경변수를
```suggestion
          webhook: ${{ secrets.DISCORD_WEBHOOK_BOT }}
```
로 바꾸는게 어때요?

---

559866add25ce31129cdf886c9af879fc4b9daa2 수정했습니다.